instrumentation/rack/example/trace_demonstration.rb

```diff
@@ -11,7 +11,7 @@
 
 ENV['OTEL_TRACES_EXPORTER'] = 'console'
 OpenTelemetry::SDK.configure do |c|
-  c.use 'OpenTelemetry::Instrumentation::Rack'
+  c.use 'OpenTelemetry::Instrumentation::Rack', { response_propagator: [OpenTelemetry::Trace::Propagation::TraceContext::ResponseTextMapPropagator.new] }
```

Comment:
Would rec. moving this to a separate example and noting that in `example/README`

Reply:
Since there's no README in the `example/` directory, I simply moved this to another file and added a comment explaining what's going on here.